bazel_tools/ghci-template.sh

```diff
@@ -4,6 +4,11 @@
 
 ENV_FILE=$1
 ARGS_FILE=$2
-RULES_HASKELL_EXEC_ROOT=$(dirname $(readlink ${BUILD_WORKSPACE_DIRECTORY}/bazel-out))/
+RULES_HASKELL_EXEC_ROOT=bazel-$(basename ${BUILD_WORKSPACE_DIRECTORY})/
```

Comment:
Was this a necessary change or just a simplification?

Reply:
That was a necessary change, see [commit message](https://github.com/digital-asset/daml/pull/3275/commits/e12176a02206dfb553fb9ebe5883c9d0b342d227)
> Cabal package library paths are longer and overflow the command-line
length when calling `cc` on MacOS for template Haskell dependencies in
the IDE tests. Shortening the `RULES_HASKELL_EXEC_ROOT` prefix for each
`(LD_)LIBRARY_PATH` entry works around the issue.